src/services/event.js

```diff
@@ -206,29 +206,28 @@ const getFutureEvents = (allEvents) => {
  * @param {Object} filters the events and filters to use
  * @returns {Event[]} The filtered events
  */
-const getFilteredEvents = (filters) => {
-  const allEvents = filters.events;
+const getFilteredEvents = (filters, allEvents, search) => {
   let filteredEvents = [];
   let shownEvents = [];
   filteredEvents = filterbyCategory(filters, allEvents);
   if (filteredEvents === null) {
     filteredEvents = allEvents;
   }
```

Comment:
I may put this logic in the actual **filterByCategory** function. Then you can run with the assumption that you will always be working with an array as the return value.

Reply:
https://github.com/gordon-cs/gordon-360-ui/pull/1015/commits/d6fb428c5d788b0d39d7e59c72ea99d6434c12ac